site/pages/api/create-consequence-network.api.ts

```diff
@@ -71,10 +72,26 @@ const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiRespons
                 ? DISRUPTION_DETAIL_PAGE_PATH
                 : REVIEW_DISRUPTION_PAGE_PATH;
 
+        if (addAnotherConsequence) {
+            const disruption = await getDisruptionById(validatedBody.data.disruptionId, session.orgId, !!template);
```

Comment:
The `upsertConsequence` method invokes method `getDisruptionById` - could we return the disruption obtained from upsert consequence method and use it in this block?

Reply:
done